docs/runtime-contract.md

```diff
@@ -417,13 +415,13 @@ In addition to the filesystems recommended in the OCI, the following filesystems
 | `/tmp`     | MUST be Read-write.<p>SHOULD be backed by tmpfs if disk load is a concern.                                                                                       |
 | `/var/log` | MUST be a directory with write permissions for logs storage. Implementations MAY permit the creation of additional subdirectories and log rotation and renaming. |
 
-In addition, the following files may be overridden by the runtime environment to
-enable DNS resolution:
+In addition, the following constraints are specified for the overridden files
+indicated:
```

Comment:
This rewrite removes the DNS rationale. What about:

"To enable DNS resolution, the following files might be overwritten at runtime:"

Reply:
much better - done